packages/api-reference-generator/.mocharc.js

```diff
@@ -8,7 +8,7 @@ process.on('unhandledRejection', (reason, promise) => {
 })
 
 module.exports = {
-  require: ['ts-node/register/transpile-only', '@earljs/published/mocha'],
+  require: ['ts-node/register/transpile-only', require.resolve('@earljs/published/mocha')],
```

Comment:
I'm quite curious - how did you come up with this solution?

Reply:
That was actually quite trivial.

The call stack in the error clearly showed that Mocha tries to require `@earljs/published/mocha` relative to their code.
PNPM is not hoisting node_modules, so this couldn't work, because Mocha doesn't depend on Earl.

However, `api-reference-generator` does depend on Earl and from this config file we can easily resolve the path to local node_modules, so if I were to translate this diff to English — we eagerly find the path to the integration instead of lazily letting Mocha find it.